src/components/PastActivityModal.js

```diff
@@ -0,0 +1,91 @@
+import styled from 'styled-components';
+
+import { QuitButton } from '../components/Button';
+import { ModalButton } from './Button.js';
```

Comment:
You could do the imports at once and at the QuitButton u don't need the `../components `